packages/fast-component-explorer/app/explorer.tsx

```diff
@@ -321,7 +325,12 @@ class Explorer extends Foundation<
                         >
                             <Background value={neutralLayerL3} drawBackground={false}>
                                 <div className={classNames(explorer_paneTitleContainer)}>
-                                    <Heading size={HeadingSize._6}>FAST Explorer</Heading>
+                                    <Heading
```

Comment:
We discussed somehow denoting that these changes could be pre-release and not available via npm in the staging case - is that work going into this PR?

Reply:
I'm working on that as part of this PR.